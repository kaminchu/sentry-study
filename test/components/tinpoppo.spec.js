// @flow
import React from "react";
import Tinpoppo from "../../src/js/components/tinpoppo";
import renderer from "react-test-renderer";

describe("components/tinpoppo", () => {
  test("snapshot test", () => {
    const component = renderer.create(
      <Tinpoppo />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});