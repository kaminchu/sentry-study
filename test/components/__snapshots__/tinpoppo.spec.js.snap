// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/tinpoppo snapshot test 1`] = `
<pre
  className="sc-bdVaJa pwNqb"
>
      ∧　∧
  (*‘ω‘ *) 　ちんぽっぽ
   (　　　)
  　 v v 　　　
  　　　　  　　ぼいんっ
  　  川
  ( (　　) )
</pre>
`;
